# An obstacle with different position
Header header
float64[] x0
float64[] th_r

float64[] center_dyn

float64[] xd
float64[] w

float32[] a 
float32[] gamma
float32[] p

float32 sf
float32 sigma
